Features/Skills/Sources/AppCircleView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+
+public struct AppCircleView: View {
+    var didUse: Bool
+
+    public init(
+        didUse: Bool
+    ) {
+        self.didUse = didUse
+    }
+
+    public var body: some View {
+        Circle()
+            .stroke(style: StrokeStyle(lineWidth: 1, dash: [4, 1]))
+            .frame(width: 15, height: 15)
```

Comment:
Magic numbers, 15 probably should be in common constants and stroke number in local one